mob_test.go

```diff
@@ -742,6 +759,95 @@ func TestStartOnUnpushedFeatureBranch(t *testing.T) {
 
 	assertOnBranch(t, "feature1")
 	assertOutputContains(t, output, "Remote branch origin/feature1 is missing")
+	assertOutputContains(t, output, "mob start --create")
+}
+
+func TestStartOnUnpushedFeatureBranchWithUncommitedChanges(t *testing.T) {
+	output, configuration := setup(t)
+	git("checkout", "-b", "feature1")
+	assertOnBranch(t, "feature1")
+	createFile(t, "file.txt", "contentIrrelevant")
+	start(configuration)
+
+	assertOnBranch(t, "feature1")
+	assertOutputContains(t, output, "mob start --include-uncommitted-changes")
+}
+
+func TestStartCreateOnUnpushedFeatureBranch(t *testing.T) {
+	_, configuration := setup(t)
+	configuration.StartCreateRemoteBranch = true
+	git("checkout", "-b", "feature1")
+	assertOnBranch(t, "feature1")
+	start(configuration)
+
+	assertOnBranch(t, "mob/feature1")
+}
+
+func TestStartCreateOnUnpushedFeatureBranchWithUncommitedChanges(t *testing.T) {
+	output, configuration := setup(t)
+	configuration.StartCreateRemoteBranch = true
+	git("checkout", "-b", "feature1")
+	assertOnBranch(t, "feature1")
+	createFile(t, "file.txt", "contentIrrelevant")
+	start(configuration)
+
+	assertOutputContains(t, output, "To start, including uncommitted changes, use")
```

Comment:
Can we also hint the user of the creation of the branch in this case?

Reply:
done.